Mods/Alpakit/Source/Alpakit/Private/AlpakitProfile.cpp

```diff
@@ -88,14 +89,18 @@ FString FAlpakitProfile::MakeUATCommandLine() {
 
 	CommandLine += bMergeArchive ? TEXT(" -merge") : TEXT("");
 
-	for (auto [Platform, GameInfo] : PlatformGameInfo) {
+	for (auto& [Platform, GameInfo] : PlatformGameInfo) {
 		if (GameInfo.bCopyToGame) {
 			CommandLine += FString::Printf(TEXT(" -CopyToGameDirectory_%s=\"%s\""), *Platform, *GameInfo.GamePath.Path);
 		}
 		if (GameInfo.bStartGame) {
 			CommandLine += FString::Printf(TEXT(" -LaunchGame_%s=%s"), *Platform, LexToString(GameInfo.StartGameType));
 		}
+		if (GameInfo.StartGameType == EAlpakitStartGameType::CUSTOM) {
+			CommandLine += FString::Printf(TEXT(" -CustomLaunchPath_%s=\"%s\""), *Platform, *GameInfo.CustomLaunchPath);
+		}
 	}
 
+	UE_LOG(LogAlpakit, Display, TEXT("Final UAT Command Line: %s"), *CommandLine);
```

Comment:
Probably a debug log as well, the UAT command line is already logged by UE when launching UAT